tests/common/playground.rs

```diff
@@ -447,3 +520,47 @@ pub enum RetryExpectations {
     //ExpectNoOut
     AllowAny { allow_out: bool, allow_err: bool },
 }
+
+fn try_lookup_srv(addr: String) -> Vec<String> {
```

Comment:
This is almost the same as the `try_lookup_srv` in the Client, except this doesn't return a client error. I think the only way to have the function shared would be to create an external utilities crate, since this would be one of the only functions in there I'm not sure that it's worth the trouble.